draft-ietf-scitt-architecture.md

```diff
@@ -93,11 +93,16 @@ normative:
   RFC5280: PKIX
   RFC6838:
   RFC8610: CDDL
-  RFC9052: COSE
+  STD94:
```

Comment:
@henkbirkholz what is this format? Just curious why this was changed, and should we do it consistently?

Reply:
I was also not aware of this, and in case someone else is wondering, the full list can be found at https://www.rfc-editor.org/standards#IS